tests/config_test.go

```diff
@@ -356,7 +366,11 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					"password": "redhat",
 				}
 
-				tests.CreateConfigMap(configMapName, testsuite.GetTestNamespace(nil), configData)
+				_, err := kubevirt.Client().CoreV1().ConfigMaps(testsuite.GetTestNamespace(nil)).Create(context.Background(), &k8sv1.ConfigMap{
+					ObjectMeta: metav1.ObjectMeta{Name: configMapName},
+					Data:       configData,
+				}, metav1.CreateOptions{})
+				Expect(err).NotTo(HaveOccurred())
```

Comment:
Just random statistics:

```
toso@tapioca ~/s/k/kubevirt (main)> grep -rniI "NotTo.Have" tests/ | wc -l
270
toso@tapioca ~/s/k/kubevirt (main)> grep -rniI "ToNot.Have" tests/ | wc -l
2487
```

So, `ToNot` is used about 10 times more in `tests/` than `NotTo`.

Reply:
@victortoso Done sir